prow/pod-utils/decorate/podspec.go

```diff
@@ -742,7 +742,14 @@ func decorate(spec *coreapi.PodSpec, pj *prowapi.ProwJob, rawEnv map[string]stri
 		wrappers = append(wrappers, *wrapperOptions)
 	}
 
-	sidecar, err := Sidecar(pj.Spec.DecorationConfig, blobStorageOptions, blobStorageMounts, logMount, outputMount, encodedJobSpec, !RequirePassingEntries, !IgnoreInterrupts, secretVolumeMounts, wrappers...)
+	var ignoreInterrupts bool
+	if pj.Spec.DecorationConfig.IgnoreInterrupts == nil { // if nil, set it to false
+		ignoreInterrupts = false
+	} else {
+		ignoreInterrupts = *pj.Spec.DecorationConfig.IgnoreInterrupts
+	}
```

Comment:
nit: This is a bit simpler
```suggestion
	ignoreInterrupts := pj.Spec.DecorationConfig.IgnoreInterrupts != nil && *pj.Spec.DecorationConfig.IgnoreInterrupts
```

Reply:
Updated it